helm_deploy/prison-estate/templates/secrets.yaml

```diff
@@ -4,4 +4,6 @@ metadata:
   name: {{ template "app.name" . }}
 type: Opaque
 data:
-  APPINSIGHTS_INSTRUMENTATIONKEY: {{ .Values.secrets.APPINSIGHTS_INSTRUMENTATIONKEY | b64enc | quote }}
+  {{- range $key, $value := .Values.secrets }}
```

Comment:
so does this mean the name of secret in AWS Secrets will be used as the environment variable name?

Reply:
yes, one of Matt's recent improvements

Reply:
although, good spot - that's not what I want :man_facepalming:

Reply:
I'll have to change the variable name

Reply:
ah it's fine, here it just can map :sweat: